resources/js/router/views/DashboardAdmin/index.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+	<div class="container justify-content-center">
```

Comment:
You don't need the class of `justify-content-center` here. It's not doing anything since this item is not set to `display:flex`. 👍